packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -203,7 +172,7 @@ class _SliverBlockElement extends RenderObjectElement {
       // block widget.)
       for (int index in _childElements.keys.toList()) {
         Element newChild;
-        renderObject._rebuild(index, () {
+        renderObject.allowAdditionsFor(index, () {
```

Comment:
i would feel more comfortable if we buried this call in the SliverMultiBoxAdaptorWidget class somehow